software-development-tfs/2025-09/index.html

```diff
@@ -29,7 +29,7 @@ <h3 class="alumni-list__title">December 2025</h3>
                 <div
                     id="alumni-list__cards"
                     class="alumni-list__cards grid"
-                    data-json-file="../data/25-12-softdev-tfs.json"
+                    data-json-file="../../data/25-12-softdev-tfs.json"
```

Comment:
Same here with the file name, we want consistency, if it's in the bootcamp folder already then the filename can be simplified